### PR TITLE
Update contrib to 1.13.0-rc.3

### PR DESCRIPTION
**Pull request by yaron2**
Updates to latest RC of components-contrib.

### CODE REVIEW

**Review by JoshVanL — APPROVED** [review]

**Review by ItalyPaleAle — APPROVED** [review]

**Comment by berndverst** [comment]
Make it `v1.13.0-rc.3` please

**Review by berndverst — DISMISSED** [review]
could you make it `v1.13.0-rc.3` please?

File: go.mod [hunk 1]
```diff
@@ -12,7 +12,7 @@ require (
 	github.com/argoproj/argo-rollouts v1.4.1
 	github.com/cenkalti/backoff/v4 v4.2.1
 	github.com/cloudevents/sdk-go/v2 v2.14.0
-	github.com/dapr/components-contrib v1.13.0-rc.1
+	github.com/dapr/components-contrib v1.13.0-rc.2
```
Comment (berndverst) [inline go.mod:15]: v1.13.0-rc.2 is no more. Long live `v1.13.0-rc.3`

Comment (yaron2) [reply go.mod]: Updated

**Review by artursouza — APPROVED** [review]

**Comment by berndverst** [comment]
Thanks!

**Pull request merged**